sunspot/sunspot.gemspec

```diff
@@ -30,7 +30,9 @@ Gem::Specification.new do |s|
   s.add_dependency 'rsolr', '>= 1.1.1', '< 3'
   s.add_dependency 'pr_geohash', '~>1.0'
 
-  s.add_development_dependency 'rspec', '~> 3.6.0'
+  s.add_development_dependency 'rake', '< 12.3'
```

Comment:
@serggl what's the rationale for this locking on preventing `rake` to move beyond 12.3? There are currently known rake security issues and we might want to loosen this constraint.

Reply:
Honestly, I don't recall exact purpose since it was more than 2 years ago. Perhaps some conflicts between dependencies.
There should not be a security concern here, since it is a development dependency. But worth bumping it to the latest of course, via PR, @brutuscat is you don't mind